src/booking-list/components/CalendarItems.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import { IUICalendarItems } from 'src/booking-list/types';
+
+export const CalendarItems = ({ datesRidesForUI }: IUICalendarItems) => {
+  const items = datesRidesForUI.map((item, index) => {
+    return (
+      <div className={'calendar'} key={index}>
```

Comment:
using index as a key should be the last resort, see https://reactjs.org/docs/lists-and-keys.html#:~:text=We%20don't%20recommend%20using,cause%20issues%20with%20component%20state.. Usually we use some unique id instead. In this case we could use booking startTime as it should be unique for a particular user.

Reply:
fixed